docs/001_develop/03_client-capabilities/011_criteria/index.mdx

```diff
@@ -1,11 +1,42 @@
 ---
-title: 'Criteria'
-sidebar_label: 'Criteria'
-id: client-criteria
-keywords: [crieria]
+title: 'Foundation Criteria'
+sidebar_label: 'Foundation Criteria'
+id: foundation-criteria
+keywords: [web, criteria, foundation criteria, frontend, ui]
 tags:
-- criteria
-sidebar_position: 11
+  - web
+  - criteria
+  - foundation criteria
+  - frontend
+  - ui
```

Comment:
Can you add more tags and keywords to all these pages (groovy etc.). Can use AI to help I find. We want to have more keywords to aid searchability in docs